server/config/config.toml

```diff
@@ -0,0 +1,589 @@
+# This is a TOML config file.
```

Comment:
is this directory needed??

Reply:
huh strange might be auto gen from some tests maybe, gonna remove it